src/main/java/de/rabea/HttpVerb.java

```diff
@@ -0,0 +1,33 @@
+package de.rabea;
+
+public enum HttpVerb {
+    GET,
+    POST,
+    PUT,
+    OPTIONS,
+    HEAD;
+
+    public static HttpVerb convert(String word) {
+        if (word.equals("GET")) {
+            return GET;
+        } else if (word.equals("POST")) {
+            return POST;
+        } else if (word.equals("PUT")) {
+            return PUT;
+        } else if (word.equals("OPTIONS")) {
+            return OPTIONS;
+        } else if (word.equals("HEAD")) {
+            return HEAD;
+        }
+        return null;
+    }
+
+    public static boolean isExisting(HttpVerb verb) {
```

Comment:
Not sure this is needed. The main point of enum is that there is a constrained amount of values.